tests/integration/test_fim/test_files/test_max_files_per_second/test_max_files_per_second.py

```diff
@@ -55,14 +59,48 @@ def test_max_files_per_second(get_configuration, configure_environment, restart_
     Raises:
         TimeoutError: If an expected event couldn't be captured.
     """
+    scheduled = get_configuration['metadata']['fim_mode'] == 'scheduled'
+
+    if inode_collision is True and scheduled is False:
+        pytest.skip("realtime and whodata modes do not verify inode collisions")
+
     # Create the files in an empty folder to check realtime and whodata.
     for i in range(n_files_to_create):
         fim.create_file(fim.REGULAR, test_directories[0], f'test_{i}', content='')
 
     extra_timeout = n_files_to_create / max_files_per_second
 
-    scheduled = get_configuration['metadata']['fim_mode'] == 'scheduled'
-    fim.check_time_travel(scheduled)
+    fim.check_time_travel(scheduled, monitor=wazuh_log_monitor,
+                          timeout=global_parameters.default_timeout + extra_timeout)
+
+    try:
+        wazuh_log_monitor.start(timeout=global_parameters.default_timeout + extra_timeout,
+                                callback=fim.callback_detect_max_files_per_second)
+    except TimeoutError as e:
+        if get_configuration['metadata']['max_files_per_sec'] == 0:
+            pass
+        else:
+            raise e
+
+    if scheduled and get_configuration['metadata']['max_files_per_sec'] != 0:
+        # Walk to the end of the scan
+        wazuh_log_monitor.start(timeout=global_parameters.default_timeout + extra_timeout,
+                                callback=fim.callback_detect_end_scan)
+
+    # Remove all files
+    for i in range(n_files_to_create):
+        fim.delete_file(test_directories[0], f'test_{i}')
+
+    if inode_collision is False:
+        return  # Done testing this case
+
+    # Create the files again changing all inodes
+    fim.create_file(fim.REGULAR, test_directories[0], 'test', content='')
+    for i in range(n_files_to_create):
+        fim.create_file(fim.REGULAR, test_directories[0], f'test_{i}', content='')
+
+    fim.check_time_travel(scheduled, monitor=wazuh_log_monitor,
+                          timeout=global_parameters.default_timeout + extra_timeout)
```

Comment:
```suggestion
    for i in range(n_files_to_create):
        fim.create_file(fim.REGULAR, test_directories[0], f'test_{i}', content='')

    fim.check_time_travel(scheduled, monitor=wazuh_log_monitor,
                          timeout=global_parameters.default_timeout + extra_timeout + 5)
```
The `check_time_travel` timeout needs to be increased. In my test environment, the test has failed because it couldn't detect the `end scan` event.

Reply:
This seems to be a problem of resources on the VM that is running the test, the purpose of the `global_parameters.default_timeout` is to add time as required instead of hardcoding timeout values. The test works fine in my local test environment and [has run correctly in Jenkins](https://github.com/wazuh/wazuh/pull/8803/checks?check_run_id=2683502555)